src/components/BadgeSlider/index.tsx

```diff
@@ -11,16 +21,98 @@ export interface IBadge {
   title: string;
 }
 
+const ITEMS_PER_PAGE = 3;
+
 interface Props {
   items: IBadge[];
 }
 
 const BadgeSlider: React.FC<Props> = ({ items }) => {
+  const [currentSlide, setCurrentSlide] = useState(0);
+  const totalPages = Math.ceil(items.length / ITEMS_PER_PAGE);
+  const slides = chunk(items, ITEMS_PER_PAGE);
+  const hideControls = items.length === 0 || items.length === ITEMS_PER_PAGE;
+
   return (
-    <>
-      <DesktopSlider items={items} />
-      <MobileSlider items={items} />
-    </>
+    <CarouselProvider
+      naturalSlideWidth={500}
+      naturalSlideHeight={200}
+      isIntrinsicHeight
+      totalSlides={totalPages}
+      currentSlide={currentSlide}
+      className="relative max-w-8xl mx-auto"
+    >
+      <Slider>
+        {slides.map((slide, index) => (
+          <Slide
+            key={index}
+            index={index}
+            innerClassName="grid grid-cols-1 gap-12 justify-items-center lg:px-20 lg:grid-cols-3"
+          >
+            {slide.map((badge: IBadge) => (
+              <Badge
+                key={badge.title}
+                imageSrc={badge.src}
+                imageAlt={badge.alt}
+                title={badge.title}
+                backgroundColor={badge.backgroundColor}
+                text={badge.text}
+                className="mx-auto"
+              />
+            ))}
+          </Slide>
+        ))}
+      </Slider>
+      {!hideControls && (
+        <>
+          <ButtonBack
+            className={classnames(
+              'hidden lg:block absolute left-0 top-0 h-full'
```

Comment:
Why you need `classnames` function here?
Aren't you able to `className="...."`?